<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 249, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainWindowFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(50, 50, 50, 1), stop: 1 rgba(128, 128, 128, 1)
);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="defenitionsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/settingsWhiteMode.png</normaloff>../images/settingsWhiteMode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="recordingLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 13pt &quot;Arial&quot;;
background: none;</string>
     </property>
     <property name="text">
      <string>Not recording</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>265</y>
       <width>500</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>500</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>295</y>
       <width>500</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>500</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>500</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="microphoneButton">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>70</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/microphoneOff.png</normaloff>../images/microphoneOff.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="logoButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="gClave">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>235</y>
       <width>50</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/gClave.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearSheetButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;
border: none;
color: white;
font: 13pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>Clear Sheet</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="exportButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(25, 81, 68, 1), stop: 1 rgba(57, 148, 127, 1)
);

border-radius: 10%;</string>
     </property>
     <property name="text">
      <string>Export Sheet</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <zorder>defenitionsButton</zorder>
    <zorder>recordingLabel</zorder>
    <zorder>line4</zorder>
    <zorder>line3</zorder>
    <zorder>line2</zorder>
    <zorder>line1</zorder>
    <zorder>line5</zorder>
    <zorder>microphoneButton</zorder>
    <zorder>logoButton</zorder>
    <zorder>clearSheetButton</zorder>
    <zorder>exportButton</zorder>
    <zorder>gClave</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
